src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -139,8 +139,9 @@ void GcsNodeManager::HandleCheckAlive(rpc::CheckAliveRequest request,
                                       rpc::CheckAliveReply *reply,
                                       rpc::SendReplyCallback send_reply_callback) {
   reply->set_ray_version(kRayVersion);
-  for (const auto &addr : request.raylet_address()) {
-    bool is_alive = node_map_.right.count(addr) != 0;
```

Comment:
I think node_map_ is not used anymore after this PR. could you remove it as well